docs/themes/oso-tailwind/layouts/partials/head.html

```diff
@@ -2,11 +2,16 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>{{ block "title" . }}
+        {{- if .Params.metaTitle -}}
+        {{- replace .Params.metaTitle "LANG" .Page.Language.LanguageName }} - {{ .Site.Title -}}
```

Comment:
I wonder whether we actually want the end of this " - Oso Documentation". Is this good for SEO?

Reply:
(Same for non-meta title changes)

Reply:
I think it's fine but I'll check with Stephie